app/code/Magento/Captcha/Model/DefaultModel.php

```diff
@@ -314,7 +314,7 @@ public function isCorrect($word)
         $storedWord = $this->getWord();
         $this->clearWord();
 
-        if (!$word || !$storedWord) {
+        if ($word != $storedWord) {
```

Comment:
It will brake logic that we have below (when case insensitive as example). Probably some unit tests will fail.

Please re-check it

Reply:
i change that logic, it will return true when 
$storedWord === null and !$word ()